compatibilityTests/src/test/scala/org/apache/spark/sql/SnappyStattisticsCollectionSuite.scala

```diff
@@ -1,26 +0,0 @@
-/*
- * Licensed to the Apache Software Foundation (ASF) under one or more
- * contributor license agreements.  See the NOTICE file distributed with
- * this work for additional information regarding copyright ownership.
- * The ASF licenses this file to You under the Apache License, Version 2.0
- * (the "License"); you may not use this file except in compliance with
- * the License.  You may obtain a copy of the License at
- *
- *    http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package org.apache.spark.sql
-
-import org.apache.spark.sql.test.SharedSnappySessionContext
-
-class SnappyStattisticsCollectionSuite
-    extends StatisticsCollectionSuite
-        with SharedSnappySessionContext {
-
-}
```

Comment:
Do all tests in this suite fail? Won't it be better to keep the class around and mark all as ignored so that we do not miss fixing these.

Reply:
There are duplicates. The 2nd one got created by mistake. Hence removing it
1: SnappySta**tt**isticsCollectionSuite and 
2: SnappyStatisticsCollectionSuite